### DISCUSSION TITLE
Running multitabs

### DISCUSSION THREAD

**Discussion by Arissm2** [community/community#136606] [Other Feature Feedback, Questions, & Ideas]
I want to ask, I'm starting to seriously work on cryptocurrency airdrops using a desktop PC, 

To open many tabs at the same time,  need a large VGA? Or only a powerful CPU and a lot of RAM are enough? Does using multitabs simultaneously require an external GPU? Ex running bot telegram same time 

Thank you

**Comment by elsycharles** [comment]
Hi,
When opening many tabs simultaneously, the main factors that determine performance are CPU power and RAM.
So focus on a powerful CPU and plenty of RAM but the external GPU is not really required for general multitabbing or running bots unless you are doing graphically intensive tasks.
Hope that help you

**Reply by Arissm2** [reply]
Thank you for the enlightenment, it's very helpful
